chpl/chpl-service/src/main/java/gov/healthit/chpl/web/controller/CertifiedProductController.java

```diff
@@ -697,7 +697,12 @@ public void streamSEDDetailsDocumentContents(final HttpServletResponse response)
             notes = "Pending certified products are created via CSV file upload and are left in the 'pending' state "
                     + " until validated and approved by an appropriate ACB administrator.")
     @RequestMapping(value = "/pending", method = RequestMethod.GET, produces = "application/json; charset=utf-8")
-    public @ResponseBody PendingCertifiedProductResults getPendingCertifiedProducts() throws EntityRetrievalException {
+    public @ResponseBody PendingCertifiedProductResults getPendingCertifiedProducts() throws EntityRetrievalException, AccessDeniedException {
+    	
+    	if(Util.isUserRoleAdmin()){
```

Comment:
If someone is both ROLE_ADMIN and ROLE_ACB they should be able to see the Pending things that are associated with whatever ACBs they have. I think this would restrict them from viewing those